virtual-machine/encrypt-vm/encrypt-windows-vm.ps1

```diff
@@ -6,7 +6,7 @@ $rgName = "myResourceGroup"
 #Region
 $location = "East US"
 #Password to place w/in the KeyVault
-$securePassword = ConvertTo-SecureString -String "Password@16digits" -AsPlainText -Force
+$securePassword = ConvertTo-SecureString -String "P@ssword!" -AsPlainText -Force
```

Comment:
Only use user-supplied or securely randomly generated passwords in example scripts.

You can do this in a cross-platform way by just using `[guid]::NewGuid()`.